src/assets/Azure.Core.Shared/ContentTypeUtilities.cs

```diff
@@ -8,13 +8,7 @@
 
 namespace Azure.Core.Pipeline
 {
-#if !TESTFRAMEWORK
     internal static class ContentTypeUtilities
-#else
-#pragma warning disable SA1649 // File name should match first type name
-    internal static class TestFrameworkContentTypeUtilities
-#pragma warning restore SA1649 // File name should match first type name
-#endif
```

Comment:
Is this no longer an issue?

Reply:
Nope, we did a bit of TestFramework refactoring.